src/schemas/searchCompany.schema.ts

```diff
@@ -0,0 +1,14 @@
+import * as Joi from '@hapi/joi'
+
+const formSchema = Joi.object({
+  companyNumber: Joi.string()
+    .required()
+    .max(8)
+    .messages({
+      'string.empty': 'You must enter a valid Company Number',
```

Comment:
Maybe move the error into a const at the top of the file to avoid redefining it multiple times

Reply:
Moved error message to const for string.max and any.required! There is a different error message for string.empty so its just in a string.